access/resource_secret_scope.go

```diff
@@ -123,11 +124,13 @@ func ResourceSecretScope() *schema.Resource {
 		s["name"].ValidateFunc = validScope
 		s["initial_manage_principal"].ForceNew = true
 		s["keyvault_metadata"].ForceNew = true
+
 		return s
 	})
-	return common.Resource{
+	resource := common.Resource{
```

Comment:
probably make CustomizeDiff package-local function and add unit tests for corner cases